modules/selenium/src/test/java/org/testcontainers/junit5/GH5116FirstTest.java

```diff
@@ -0,0 +1,20 @@
+package org.testcontainers.junit5;
+
+import org.junit.jupiter.api.Test;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.is;
+import static org.hamcrest.Matchers.notNullValue;
+
+class GH5116FirstTest extends BaseTest {
```

Comment:
given `testContainerIsAlreadyConfiguredWhenUsedMultipleTimes`, do you still think we need these tests? Browser containers are expensive, would be nice to avoid having to start them if not needed (although I appreciate the full reproducer according to the original issue 💯)